surfinpy/bulk_mu_vs_t.py

```diff
@@ -115,9 +114,11 @@ def evaluate_phases(data, bulk, x, y,
     S = np.array([])
     new_data_svib = 0
```

Comment:
This variable no longer matches the one beneath it. So some tutorials fail when, if bulk.entropy is False, there is no new_bulk_svib to pass to calculate_bulk_energy.